modules/installing-ocp-agent-boot.adoc

```diff
@@ -116,41 +116,41 @@ platform:
 +
 [source,yaml]
 ----
-  cat > agent-config.yaml << EOF
-  apiVersion: v1alpha1
-  kind: AgentConfig
-  metadata:
-    name: sno-cluster
-  rendezvousIP: 192.168.111.80 <1>
-  hosts: <2>
-    - hostname: master-0 <3>
+cat > agent-config.yaml << EOF
```

Comment:
So this was already like this before your update, but since `cat` is a command to run from the command line, there should be a "$" prompt in front, and should technically be `[source,terminal]` instead of yaml. Just in case you care to make those fixes while you're here - no worries if not!